rules/xcodeproj.bzl

```diff
@@ -63,6 +63,29 @@ def _xcodeproj_aspect_collect_hmap_paths(deps, target, ctx):
                 hmap_paths.append(relative_path)
     return hmap_paths
 
+def _targeted_device_family(ctx):
```

Comment:
does it potentially make sense to use `_ui_device_family_plist_value` from https://github.com/bazelbuild/rules_apple/blob/37a2e525fa048cb9b7bf2ce776804f622c5a4262/apple/internal/platform_support.bzl#L58-L70 ?

Reply:
Thanks for the pointer!